infra/base-images/base-builder/Dockerfile

```diff
@@ -209,4 +209,8 @@ ENV CCACHE_DIR /ccache/cache
 ENV CCACHE_COMPILERCHECK none
 ENV CCACHE_COMPILERTYPE clang
 
+COPY indexer /opt/indexer
```

Comment:
let's pull this from the clusterfuzz-builds bucket for  now via `ADD`.

Reply:
Some of it is scripts I copied though, can I commit those, they aren't binaries?

Reply:
discussed offline -- they should be good to go.